test/scenarios/cloudnative/basic/README.md

```diff
@@ -0,0 +1,58 @@
+# Prerequisites
+k8s cluster
+
+# Setup
+CloudNative deployment with CSI driver
+
+# Goals
+Verify that the operator is working as expected.
+
+Scenarios:
+### Install
+- DTO installed
+- sample application Deployment installed
+- check if oneagent started
+- check if Dynakube phase changed to Running
+- sample application restarted
+- check if oneagent init container is injected
+- check if oneagents are able to communicate with the cluster
+
+### Upgrade
+Verify that 0.9.1 released version can be installed from GitHub
+
+- DTO "v0.9.1" installed (from GitHub)
+- sample application Deployment installed
+- check if oneagent started
+- check if Dynakube phase changed to Running
+- sample application restarted
+- check if oneagent init container is injected
+- check if oneagents are able to communicate with the cluster
+
+### CodeModules
+Verify that the storage in the CSI driver directory does not increase when there are multiple tenants and pods which are monitored.
+
+- DTO installed
+- sample application Deployment installed
+- check if oneagent started
```

Comment:
```suggestion
- check if OneAgent started
```

Reply:
AFAIK OneAgent is a kind of brand name or something so it should be written in capital letters. This comment applies to all uses of `oneagent` in this PR.

Reply:
not official documentation, so unnecessary in my opinion